doc/man1/ocsp.pod

```diff
@@ -299,16 +300,17 @@ digest used by subsequent certificate identifiers.
 
 =item B<-index indexfile>
 
-B<indexfile> is a text index file in B<ca> format containing certificate revocation
-information.
+B<Indexfile> is a text index file in B<ca> format containing certificate
```

Comment:
Not sure about this one. This is a parameter name, so surely it should be a lower case to match how it is written above?